lib/shipit/engine.rb

```diff
@@ -38,6 +38,8 @@ class Engine < ::Rails::Engine
         end
       end
 
+      app.config.middleware.insert_after(::Rack::Runtime, Shipit::SameSiteCookieMiddleware)
```

Comment:
I'm always worried with `insert_after/before`, as middleware stacks are kind of a moving target. Do we need to be that specific about it's order?

Reply:
My understanding is that this middleware needs to be at least added before `ActionDispatch::Cookies`. This is the PR that introduced this change in shopify_app: https://github.com/Shopify/shopify_app/pull/898